bundles/CoreBundle/src/Migrations/Version20220829132224.php

```diff
@@ -75,8 +75,8 @@ private function migrateCustomViewsPerspectives(string $fileName, string $scope)
     private function migrateWeb2PrintSettings(): void
     {
         $configs = $this->loadLegacyConfigs('web2print.php');
-        if (count($configs) > 0) {
-            $web2PrintConfigs = Config::getWeb2PrintConfig();
+        if (count($configs) > 0 && class_exists('Pimcore\Bundle\WebToPrintBundle\Config')) {
```

Comment:
I am not to sure about changing this migration

Reply:
I'd wrap the entire `function up()` with a IF block that checks if it exists under the SettingStore and in case "skips" everything.
The `down()` is empty anyways.

Reply:
I am not sure how this check should look like. Do you have an example?

Reply:
```
public function up(Schema $schema): void
{
  if  (!SettingsStore::get('web_to_print', 'pimcore_web_to_print'){
      $this->migrateCommonConfigurations('predefined-properties.php', 'pimcore_predefined_properties');
       ...
       $this->migrateWeb2PrintSettings();
  }
}
```
when that exists, it should do nothing, assuming the migration was already done in the past or doesn't need because there are no more legacy config files.